graveler/ref/manager.go

```diff
@@ -41,7 +42,17 @@ func (m *Manager) GetRepository(ctx context.Context, repositoryID graveler.Repos
 	return repository.(*graveler.Repository), nil
 }
 
-func (m *Manager) CreateRepository(ctx context.Context, repositoryID graveler.RepositoryID, repository graveler.Repository, branch graveler.Branch) error {
+const (
+	firstCommitMsg = "lakeFS repository created!"
```

Comment:
1. Public under graveler so we can verify when testing.
2. Consider "Initial repository created" or something without lakeFS.

Reply:
Done. "Initial" for repository is kinda weird, so kept it simple.